test/C/src/federated/FeedbackDelay3.lf

```diff
@@ -0,0 +1,40 @@
+target C {
```

Comment:
It might be nice for these new `FeedbackDelay` tests to have a short comment explaining what makes them different from each other or what different bugs they catch. This is because `FeedbackDelay` is one of the most complicated tests, so it can be hard for me to see what is going on just by looking at it.

Reply:
Good suggestion.  Comments added in commit #7408a89d5.